docs/setup/minikube/getting-started-minikube.md

```diff
@@ -8,19 +8,20 @@ Follow this step-by-step guide to set up Nuclio on [Minikube](https://github.com
 - [Prepare Minikube](#prepare-minikube)
```

Comment:
this is going to be confliced due to 
https://github.com/nuclio/nuclio/pull/1872

so please wait until it is merged and resolve correspondingly

Reply:
The Minikube doc-update PR #1872 has now been merged to `development` and I've now merged it manually to the current PR branch, including replacing "Invoke" with "Test" — see commit 712a239.